quackfactory/cogs/session.py

```diff
@@ -0,0 +1,28 @@
+import discord
+
+class Session(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+@client.event

```

Comment:
```suggestion
@commands.Cog.listener()
```
You are using the wrong bot object for this to work as a cog.

it has to be the `discord.commands` bot instance. Please refer to [reactions.py](https://github.com/silobusters/quackfactory/blob/master/quackfactory/cogs/reactions.py)

Reply:
Alright.